core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -619,6 +619,9 @@ private[deploy] class Master(
         case e: Exception => logInfo("Worker " + worker.id + " had exception on reconnect")
       }
     }
+
+    // In case of zero workers and apps, we can complete recovery.
+    if (canCompleteRecovery) { completeRecovery() }
```

Comment:
super nit:
```suggestion
    if (canCompleteRecovery) {
      completeRecovery()
    }
```

Not sure why this is the style in rest of the class though :-(

Reply:
Ya, I agree. Sure, let me change.